cms/envs/common.py

```diff
@@ -571,6 +571,16 @@
     # .. toggle_creation_date: 2024-03-14
     # .. toggle_tickets: https://github.com/openedx/edx-platform/pull/34173
     'ENABLE_HOME_PAGE_COURSE_API_V2': True,
+
+    # .. toggle_name: FEATURES['ENABLE_BLAKE2B_HASH_FN']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Enables the memcache to use the blake2b hash algorithm instead of depreciated md4 for keys
+    #   exceeding 250 characters
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2024-04-02
```

Comment:
I think we should mention somewhere that this should be a short-lived toggle. Maybe only until the sycamore release.

Reply:
And do we need to define this twice?

Reply:
Are you talking about comments? Yes, we don't need to add the comments again in the CMS